src/set_theory/cardinal/finite.lean

```diff
@@ -34,6 +34,64 @@ lemma card_eq_fintype_card [fintype α] : nat.card α = fintype.card α := mk_to
 @[simp]
 lemma card_eq_zero_of_infinite [infinite α] : nat.card α = 0 := mk_to_nat_of_infinite
 
+lemma card_eq_of_equiv (f : α ≃ β) : nat.card α = nat.card β :=
+begin
+  have : ulift α ≃ ulift β := (equiv.ulift.trans f).trans equiv.ulift.symm,
+  have := cardinal.to_nat_congr this,
+  simpa using this,
+end
```

Comment:
This is the same as `cardinal.to_nat_congr`. Should we call it `nat.card_congr`?
```suggestion
cardinal.to_nat_congr f
```